### PR TITLE
refactor: implement intro modal design

### PR DESCRIPTION
**Pull request by fbwoolf**
## Description

This PR implements new designs in the intro (install) modal.

image

For details refer to issue: https://github.com/blockstack/stacks-wallet-web/issues/1071

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] API reference/documentation update
- [x] Other

## Does this introduce a breaking change?
No.

## Are documentation updates required?
No.

## Testing
- Verify modal still works as expected

### CODE REVIEW

**Review by jasperjansz — APPROVED** [review]

File: packages/connect-ui/src/components/modal/modal.tsx [hunk 1]
```diff
@@ -26,61 +27,65 @@ export class Modal {
...
+              <div class="button-container">
+                <button
+                  class="button"
+                  onClick={() => {
+                    this.handleDownloadPath(browser);
+                  }}
                 >
-                  How it works
-                </span>
+                  <span>Download Stacks Wallet</span>
```
Comment (kyranjamie) [inline packages/connect-ui/src/components/modal/modal.tsx:78]: Why is the `span` needed here?

File: packages/connect-ui/src/components/modal/assets/close-icon.tsx [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+      stroke-width="1.5"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+    />
+    <path
+      d="M17 7L7 17"
+      stroke="#242629"
+      stroke-width="1.5"
+      stroke-linecap="round"
+      stroke-linejoin="round"
```
Comment (kyranjamie) [inline packages/connect-ui/src/components/modal/assets/close-icon.tsx:24]: Are you sure this doesn't need to be converted to jsx-friendly SVG https://svgtojsx.com/ ?

Comment (fbwoolf) [reply packages/connect-ui/src/components/modal/assets/close-icon.tsx:24]: This was already here and just a reformat, but I can definitely convert it.

**Review by kyranjamie — APPROVED** [review]

File: packages/connect-ui/src/components/modal/assets/close-icon.tsx (hunk 2 above)
Comment (fbwoolf) [reply packages/connect-ui/src/components/modal/assets/close-icon.tsx:24]: This is stenciljs not React.

Comment (aulneau) [reply packages/connect-ui/src/components/modal/assets/close-icon.tsx:24]: stencil uses jsx (see the filename is .tsx)

Comment (aulneau) [reply packages/connect-ui/src/components/modal/assets/close-icon.tsx:24]: https://stenciljs.com/docs/templating-jsx

Comment (fbwoolf) [reply packages/connect-ui/src/components/modal/assets/close-icon.tsx:24]: Why doesn't the file recognize `strokeWidth`? I get an error warning if I use the jsx converter. I'll look into it.

File: packages/connect-ui/src/components/modal/modal.tsx (hunk 1 above)
Comment (fbwoolf) [reply packages/connect-ui/src/components/modal/modal.tsx:78]: Good question, I just changed the text inside but def think can remove.

File: packages/connect-ui/src/components/modal/assets/close-icon.tsx (hunk 2 above)
Comment (fbwoolf) [reply packages/connect-ui/src/components/modal/assets/close-icon.tsx:24]: Just for ref, ended up implementing this, which seems to be working and the proper way to do it with stenciljs: https://stenciljs.com/docs/local-assets

**Pull request merged**